SETUP.md

```diff
@@ -113,242 +111,6 @@
 
 **NEXT:** if all of the containers show a `Up (healthy)` or `Exit 0` state, then you're ready to start the workshop!
 
-### Instructor-led workshop setup (for trainees)
```

Comment:
:ok_hand: 

Ideally we should not need to replace these instructions. If we decide to resume offering instructor-led training, those workshops would have slides and the only instruction trainees _should_ need is to download a new .envrc file (instead of the default one provided with the workshop).